### PR TITLE
Arbitrary summary HTML

### PR DESCRIPTION
**Pull request by josenavas**
~~Depends on https://github.com/biocore/qiita/pull/2136 so review/merge that one first.~~

Modifies the GUI for the artifact summary to enable arbitrary HTMLs.

### CODE REVIEW

**Comment by antgonza** [comment]
@josenavas, could you fix conflicts? Thanks!

**Comment by josenavas** [comment]
Sure - done!

**Comment by josenavas** [comment]
This is ready for review

File: qiita_pet/handlers/artifact_handlers/base_handlers.py [hunk 1]
```diff
@@ -108,8 +108,7 @@ def artifact_summary_get_request(user, artifact_id):
 
     # Check if the HTML summary exists
     if summary:
-        with open(summary[1]) as f:
-            summary = f.read()
+        summary = relpath(summary[1], qiita_config.base_data_dir)
```
Comment (antgonza) [inline qiita_pet/handlers/artifact_handlers/base_handlers.py]: Could you add a comment about magic [1]?

Comment (josenavas) [inline qiita_pet/handlers/artifact_handlers/base_handlers.py]: Added

**Pull request merged**